{
    "Information": {
        "Category": null, 
        "CAPEC": null, 
        "CVE": [
            {
                "url": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2008-1679", 
                "published": "2008-04-22T00:41:00.000-04:00", 
                "id": "CVE-2008-1679", 
                "modified": "2010-08-21T01:19:02.370-04:00", 
                "summary": "Multiple integer overflows in imageop.c in Python before 2.5.3 allow context-dependent attackers to cause a denial of service (crash) and possibly execute arbitrary code via crafted images that trigger heap-based buffer overflows.  NOTE: this issue is due to an incomplete fix for CVE-2007-4965."
            }
        ], 
        "CWE": [
            {
                "url": "https://cwe.mitre.org/data/definitions/189.html", 
                "id": "CWE-189", 
                "title": "Numeric Errors"
            }
        ], 
        "CPE": [
            {
                "id": "cpe:/a:python_software_foundation:python:2.5.2"
            }, 
            {
                "id": "cpe:/a:python_software_foundation:python:2.4"
            }
        ]
    }, 
    "References": {
        "SCIP": [
            {
                "url": "http://www.scip.ch/?vuldb.42078", 
                "id": 42078
            }
        ], 
        "IAVM": [], 
        "BID": [], 
        "OSVDB": [], 
        "Other": {
            "References": [
                {
                    "url": "http://www.debian.org/security/2008/dsa-1551", 
                    "vendor": "DEBIAN"
                }, 
                {
                    "url": "http://secunia.com/advisories/29889", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://www.ubuntu.com/usn/usn-632-1", 
                    "vendor": "UBUNTU"
                }, 
                {
                    "url": "http://www.novell.com/support/search.do?cmd=displayKC&docType=kc&externalId=InfoDocument-patchbuilder-readme5032900", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "http://www.mandriva.com/security/advisories?name=MDVSA-2008:164", 
                    "vendor": "MANDRIVA"
                }, 
                {
                    "url": "http://www.mandriva.com/security/advisories?name=MDVSA-2008:163", 
                    "vendor": "MANDRIVA"
                }, 
                {
                    "url": "http://www.debian.org/security/2008/dsa-1620", 
                    "vendor": "DEBIAN"
                }, 
                {
                    "url": "http://support.avaya.com/css/P8/documents/100074697", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "http://support.apple.com/kb/HT3438", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "http://slackware.com/security/viewer.php?l=slackware-security&y=2008&m=slackware-security.525289", 
                    "vendor": "SLACKWARE"
                }, 
                {
                    "url": "http://security.gentoo.org/glsa/glsa-200807-01.xml", 
                    "vendor": "GENTOO"
                }, 
                {
                    "url": "http://secunia.com/advisories/38675", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://secunia.com/advisories/33937", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://secunia.com/advisories/31687", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://secunia.com/advisories/31518", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://secunia.com/advisories/31365", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://secunia.com/advisories/31358", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://secunia.com/advisories/31255", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://secunia.com/advisories/30872", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://lists.opensuse.org/opensuse-security-announce/2008-08/msg00006.html", 
                    "vendor": "SUSE"
                }, 
                {
                    "url": "http://lists.apple.com/archives/security-announce/2009/Feb/msg00000.html", 
                    "vendor": "APPLE"
                }, 
                {
                    "url": "http://bugs.python.org/msg64682", 
                    "vendor": "MISC"
                }, 
                {
                    "url": "http://bugs.python.org/issue1179", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "https://issues.rpath.com/browse/RPL-2424", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "http://xforce.iss.net/xforce/xfdb/41958", 
                    "vendor": "XF"
                }, 
                {
                    "url": "http://wiki.rpath.com/wiki/Advisories:rPSA-2008-0149", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "http://secunia.com/advisories/29955", 
                    "vendor": "SECUNIA"
                }
            ]
        }, 
        "CertVN": []
    }, 
    "Risk": [
        {
            "CVSS v2": [
                {
                    "impact": "6.4", 
                    "access complexity": "medium", 
                    "confidentiality impact": "partial", 
                    "authentication": "none", 
                    "exploit": "8.6", 
                    "base": "6.8", 
                    "integrity impact": "partial", 
                    "availability impact": "partial", 
                    "access vector": "network"
                }
            ], 
            "severity": "Moderate", 
            "Top alert": false, 
            "Top vulnerable": false
        }
    ], 
    "Rules": {
        "Suricata": [], 
        "Snort": []
    }, 
    "Exploits": {
        "Metasploit": [], 
        "ExploitDB": [], 
        "Elliot D2": [], 
        "Saint": []
    }, 
    "vFeed": {
        "Product": "vFeed - The Correlated Vulnerability and Threat Database", 
        "Contact": "@toolswatch", 
        "Author": "NJ OUCHN", 
        "URL": "https://github.com/toolswatch/vFeed", 
        "Version": "0.6.5", 
        "id": "VFD-2008-1679"
    }, 
    "Scanners": {
        "Oval": [
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:17381", 
                "title": "USN-632-1 -- python2.4, python2.5 vulnerabilities", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:17381"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:7981", 
                "title": "DSA-1620 python2.5 -- several vulnerabilities", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:7981"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:18481", 
                "title": "DSA-1551-1 python2.4 - several vulnerabilities", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:18481"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:10583", 
                "title": "Multiple integer overflows in imageop.c in Python before 2.5.3 allow context-dependent attackers to cause a denial of service (crash) and possibly execute arbitrary code via crafted images that trigger heap-based buffer overflows.  NOTE: this issue is due to an incomplete fix for CVE-2007-4965.", 
                "class": "vulnerability", 
                "id": "oval:org.mitre.oval:def:10583"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:20188", 
                "title": "DSA-1620-1 python2.5 - several vulnerabilities", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:20188"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:8152", 
                "title": "DSA-1551 python2.4 -- several vulnerabilities", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:8152"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:7800", 
                "title": "Multiple Buffer and Integer Overflow Vulnerabilities in Python (python(1)) May Lead to a Denial of Service (DoS) or Allow Execution of Arbitrary Code", 
                "class": "vulnerability", 
                "id": "oval:org.mitre.oval:def:7800"
            }
        ], 
        "Nmap": [], 
        "OpenVas": [
            {
                "name": "Debian Security Advisory DSA 1551-1 (python2.4)", 
                "file": "deb_1551_1.nasl", 
                "family": "Debian Local Security Checks", 
                "id": "60798"
            }, 
            {
                "name": "Debian Security Advisory DSA 1620-1 (python2.5)", 
                "file": "deb_1620_1.nasl", 
                "family": "Debian Local Security Checks", 
                "id": "61372"
            }, 
            {
                "name": "Slackware Advisory SSA:2008-217-01 python ", 
                "file": "esoft_slk_ssa_2008_217_01.nasl", 
                "family": "Slackware Local Security Checks", 
                "id": "61467"
            }, 
            {
                "name": "CentOS Update for python CESA-2009:1178 centos3 i386", 
                "file": "gb_CESA-2009_1178_python_centos3_i386.nasl", 
                "family": "CentOS Local Security Checks", 
                "id": "880715"
            }, 
            {
                "name": "Mandriva Update for mysql MDVA-2012:049 (mysql)", 
                "file": "gb_mandriva_MDVSA_2008_163.nasl", 
                "family": "Mandrake Local Security Checks", 
                "id": "830740"
            }, 
            {
                "name": "Ubuntu Update for erlang vulnerability USN-624-2", 
                "file": "gb_ubuntu_USN_632_1.nasl", 
                "family": "Ubuntu Local Security Checks", 
                "id": "840343"
            }, 
            {
                "name": "Gentoo Security Advisory GLSA 200807-01 (python)", 
                "file": "glsa_200807_01.nasl", 
                "family": "Gentoo Local Security Checks", 
                "id": "61216"
            }, 
            {
                "name": "get_kb_item(ssh/login/osx_name", 
                "file": "macosx_secupd_2009-001.nasl", 
                "family": "Mac OS X Local Security Checks", 
                "id": "102026"
            }, 
            {
                "name": "Mandrake Security Advisory MDVSA-2009:003 (python)", 
                "file": "mdksa_2009_003.nasl", 
                "family": "Mandrake Local Security Checks", 
                "id": "63136"
            }, 
            {
                "name": "Mandrake Security Advisory MDVSA-2009:036 (python)", 
                "file": "mdksa_2009_036.nasl", 
                "family": "Mandrake Local Security Checks", 
                "id": "63373"
            }, 
            {
                "name": "CentOS Security Advisory CESA-2009:1178 (python)", 
                "file": "ovcesa2009_1178.nasl", 
                "family": "CentOS Local Security Checks", 
                "id": "64493"
            }, 
            {
                "name": "RedHat Security Advisory RHSA-2009:1177", 
                "file": "RHSA_2009_1177.nasl", 
                "family": "Red Hat Local Security Checks", 
                "id": "64454"
            }, 
            {
                "name": "RedHat Security Advisory RHSA-2009:1178", 
                "file": "RHSA_2009_1178.nasl", 
                "family": "Red Hat Local Security Checks", 
                "id": "64455"
            }, 
            {
                "name": "SLES10: Security update for Python", 
                "file": "sles10_python.nasl", 
                "family": "SuSE Local Security Checks", 
                "id": "65883"
            }, 
            {
                "name": "SLES9: Security update for Python", 
                "file": "sles9p5032900.nasl", 
                "family": "SuSE Local Security Checks", 
                "id": "65027"
            }
        ], 
        "Nessus": [
            {
                "name": "CentOS 3 : python (CESA-2009:1178)", 
                "file": "centos_RHSA-2009-1178.nasl", 
                "family": "CentOS Local Security Checks", 
                "id": "40394"
            }, 
            {
                "name": "Debian DSA-1551-1 : python2.4 - several vulnerabilities", 
                "file": "debian_DSA-1551.nasl", 
                "family": "Debian Local Security Checks", 
                "id": "32006"
            }, 
            {
                "name": "Debian DSA-1620-1 : python2.5 - several vulnerabilities", 
                "file": "debian_DSA-1620.nasl", 
                "family": "Debian Local Security Checks", 
                "id": "33740"
            }, 
            {
                "name": "GLSA-200807-01 : Python: Multiple integer overflows", 
                "file": "gentoo_GLSA-200807-01.nasl", 
                "family": "Gentoo Local Security Checks", 
                "id": "33421"
            }, 
            {
                "name": "Mac OS X Multiple Vulnerabilities (Security Update 2009-001)", 
                "file": "macosx_SecUpd2009-001.nasl", 
                "family": "MacOS X Local Security Checks", 
                "id": "35684"
            }, 
            {
                "name": "Mandriva Linux Security Advisory : python (MDVSA-2008:163)", 
                "file": "mandriva_MDVSA-2008-163.nasl", 
                "family": "Mandriva Local Security Checks", 
                "id": "37212"
            }, 
            {
                "name": "Oracle Linux 4 : python (ELSA-2009-1177)", 
                "file": "oraclelinux_ELSA-2009-1177.nasl", 
                "family": "Oracle Linux Local Security Checks", 
                "id": "67897"
            }, 
            {
                "name": "Oracle Linux 3 : python (ELSA-2009-1178)", 
                "file": "oraclelinux_ELSA-2009-1178.nasl", 
                "family": "Oracle Linux Local Security Checks", 
                "id": "67898"
            }, 
            {
                "name": "RHEL 4 : python (RHSA-2009:1177)", 
                "file": "redhat-RHSA-2009-1177.nasl", 
                "family": "Red Hat Local Security Checks", 
                "id": "40401"
            }, 
            {
                "name": "RHEL 3 : python (RHSA-2009:1178)", 
                "file": "redhat-RHSA-2009-1178.nasl", 
                "family": "Red Hat Local Security Checks", 
                "id": "40402"
            }, 
            {
                "name": "Scientific Linux Security Update : python for SL 3.0.x on i386/x86_64", 
                "file": "sl_20090728_python_for_SL_3_0_x.nasl", 
                "family": "Scientific Linux Local Security Checks", 
                "id": "60624"
            }, 
            {
                "name": "Scientific Linux Security Update : python for SL 4.x on i386/x86_64", 
                "file": "sl_20090728_python_for_SL_4_x.nasl", 
                "family": "Scientific Linux Local Security Checks", 
                "id": "60625"
            }, 
            {
                "name": "Slackware 10.1 / 10.2 / 11.0 / 12.0 / 12.1 / current : python (SSA:2008-217-01)", 
                "file": "Slackware_SSA_2008-217-01.nasl", 
                "family": "Slackware Local Security Checks", 
                "id": "33824"
            }, 
            {
                "name": "SuSE9 Security Update : Python (YOU Patch Number 12215)", 
                "file": "suse9_12215.nasl", 
                "family": "SuSE Local Security Checks", 
                "id": "41229"
            }, 
            {
                "name": "openSUSE Security Update : python (python-128)", 
                "file": "suse_11_0_python-080801.nasl", 
                "family": "SuSE Local Security Checks", 
                "id": "40115"
            }, 
            {
                "name": "SuSE 10 Security Update : Python (ZYPP Patch Number 5490)", 
                "file": "suse_python-5490.nasl", 
                "family": "SuSE Local Security Checks", 
                "id": "33923"
            }, 
            {
                "name": "openSUSE 10 Security Update : python (python-5491)", 
                "file": "suse_python-5491.nasl", 
                "family": "SuSE Local Security Checks", 
                "id": "33924"
            }, 
            {
                "name": "Ubuntu 6.06 LTS / 7.04 / 7.10 / 8.04 LTS : python2.4, python2.5 vulnerabilities (USN-632-1)", 
                "file": "ubuntu_USN-632-1.nasl", 
                "family": "Ubuntu Local Security Checks", 
                "id": "33807"
            }
        ]
    }, 
    "Patches": {
        "Fedora": [], 
        "HP": [], 
        "Microsoft KB": [], 
        "Mandriva": [
            {
                "url": "http://www.mandriva.com/security/advisories?name=MDVSA-2008:163", 
                "id": "MDVSA-2008:163"
            }, 
            {
                "url": "http://www.mandriva.com/security/advisories?name=MDVSA-2008:164", 
                "id": "MDVSA-2008:164"
            }
        ], 
        "Cisco": [], 
        "IBM AIX Apar": [], 
        "Suse": [
            {
                "url": "https://www.suse.com/security/cve/CVE-2008-1679.html", 
                "id": "SUSE-SR:2008:017"
            }
        ], 
        "Redhat": [
            {
                "Redhat": {
                    "oval": "oval:com.redhat.rhsa:def:20091177", 
                    "url": "https://rhn.redhat.com/errata/RHSA-2009-1177.html", 
                    "id": "RHSA-2009:1177", 
                    "title": "RHSA-2009:1177: python security update (Moderate)"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=441306", 
                    "title": "CVE-2008-1679 python: imageop module integer overflows", 
                    "id": "441306", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=442005", 
                    "title": "CVE-2008-1721 python: integer signedness error in the zlib extension module", 
                    "id": "442005", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=443810", 
                    "title": "CVE-2008-1887 python: PyString_FromStringAndSize does not check for negative size values", 
                    "id": "443810", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=454990", 
                    "title": "CVE-2008-3142 python: Multiple buffer overflows in unicode processing", 
                    "id": "454990", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455008", 
                    "title": "CVE-2008-2315 python: Multiple integer overflows in python core", 
                    "id": "455008", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455013", 
                    "title": "CVE-2008-3143 python: Multiple integer overflows discovered by Google", 
                    "id": "455013", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455018", 
                    "title": "CVE-2008-3144 python: Potential integer underflow and overflow in the PyOS_vsnprintf C API function", 
                    "id": "455018", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=469656", 
                    "title": "CVE-2008-4864 python: imageop module multiple integer overflows", 
                    "id": "469656", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=470915", 
                    "title": "CVE-2008-5031 python: stringobject, unicodeobject integer overflows", 
                    "id": "470915", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "Redhat": {
                    "oval": "oval:com.redhat.rhsa:def:20091178", 
                    "url": "https://rhn.redhat.com/errata/RHSA-2009-1178.html", 
                    "id": "RHSA-2009:1178", 
                    "title": "RHSA-2009:1178: python security update (Moderate)"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=441306", 
                    "title": "CVE-2008-1679 python: imageop module integer overflows", 
                    "id": "441306", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1178"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=443810", 
                    "title": "CVE-2008-1887 python: PyString_FromStringAndSize does not check for negative size values", 
                    "id": "443810", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1178"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=454990", 
                    "title": "CVE-2008-3142 python: Multiple buffer overflows in unicode processing", 
                    "id": "454990", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1178"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455008", 
                    "title": "CVE-2008-2315 python: Multiple integer overflows in python core", 
                    "id": "455008", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1178"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455013", 
                    "title": "CVE-2008-3143 python: Multiple integer overflows discovered by Google", 
                    "id": "455013", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1178"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455018", 
                    "title": "CVE-2008-3144 python: Potential integer underflow and overflow in the PyOS_vsnprintf C API function", 
                    "id": "455018", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1178"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=469656", 
                    "title": "CVE-2008-4864 python: imageop module multiple integer overflows", 
                    "id": "469656", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1178"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=470915", 
                    "title": "CVE-2008-5031 python: stringobject, unicodeobject integer overflows", 
                    "id": "470915", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1178"
                }
            }
        ], 
        "Gentoo": [
            {
                "url": "https://security.gentoo.org/glsa/200807-01", 
                "id": "GLSA-200807-01"
            }
        ], 
        "Microsoft Bulletins": [], 
        "Ubuntu": [
            {
                "url": "http://www.ubuntu.com/usn/USN-632-1", 
                "id": "USN-632-1"
            }
        ], 
        "Debian": [
            {
                "url": "https://security-tracker.debian.org/tracker/DSA-1551", 
                "id": "DSA-1551"
            }, 
            {
                "url": "https://security-tracker.debian.org/tracker/DSA-1620", 
                "id": "DSA-1620"
            }
        ], 
        "Vmware": []
    }
}